docs/scripts/plop.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Plop Scripts
+---
+
+# Plop [![version](https://img.shields.io/npm/v/@pota/plop-scripts.svg?label=%20)](https://npmjs.org/package/@pota/plop-scripts)
+
+A Pota wrapper command around [plop](https://github.com/plopjs/plop)
+
+## Usage
+
+> Adding to an existing project
+
+```bash
+# install the plop-scripts and cli packages
+npm install @pota/plop-scripts @pota/cli --save-dev
+
+# configure the cli package to use plop-scripts
+npm pkg set pota="@pota/plop-scripts"
```

Comment:
Would this be the only value in the `pota` field, or could there be more (where this would overwrite the old value)?

An option could be to use an array instead? `npm pkg set pota[]="@pota/plop-scripts"`

Reply:
It can both be the singular value of the `pota` field, set as just a string `"@pota/plop-scripts"` or it can be added on to that field if it is an array.

That setup guide assumes that you are installing `@pota/cli` for the first time, so you wouldn't have a `pota` field defined.